tools/simpleui/template.go

```diff
@@ -17,3 +18,12 @@ func textproto(a *anypb.Any) (string, error) {
 	}
 	return prototext.Format(m), nil
 }
+
+func parent(in string) string {
+	s := strings.Split(in, "/")
+	return strings.Join(s[:3], "/")
```

Comment:
Will `in` always have 4 elements?

```
func parent(in string) string {
	return in[:strings.LastIndex(in, "/")]
}
```

Reply:
Done!

Reply:
Ended up reverting this. We need the first 3 parts, not "everything but the last part" (there are 5 parts totally for this ID).

I think we can make a reasonable assumption about this format since it's validated at write time.